mmocr/datasets/text_spotter_dataset.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import numpy as np
+from mmdet.datasets.builder import DATASETS
+
+from .icdar_dataset import IcdarDataset
+
+
+@DATASETS.register_module()
+class TextSpotterDataset(IcdarDataset):
+
+    def _parse_ann_info(self, img_info, ann_info):
+        """Parse bbox and mask annotation.
+
+        Args:
+            ann_info (list[dict]): Annotation info of an image.
+
+        Returns:
+            dict: A dict containing the following keys: bboxes, bboxes_ignore,
+                labels, masks, masks_ignore, seg_map. "masks"  and
+                "masks_ignore" are represented by polygon boundary
+                point sequences.
+        """
+        gt_bboxes = []
+        gt_labels = []
+        gt_bboxes_ignore = []
+        gt_masks_ignore = []
+        gt_masks_ann = []
+        gt_texts = []
+        gt_ignore_texts = []
+
+        for ann in ann_info:
+            if ann.get('ignore', False):
+                continue
+            x1, y1, w, h = ann['bbox']
+            if ann['area'] <= 0 or w < 1 or h < 1:
+                continue
+            if ann['category_id'] not in self.cat_ids:
+                continue
+            bbox = [x1, y1, x1 + w, y1 + h]
+            if ann.get('iscrowd', False):
+                gt_bboxes_ignore.append(bbox)
+                gt_masks_ignore.append(ann.get(
+                    'segmentation', None))  # to float32 for latter processing
+                gt_ignore_texts.append('###')
+            else:
+                gt_bboxes.append(bbox)
+                gt_labels.append(self.cat2label[ann['category_id']])
+                gt_masks_ann.append(ann.get('segmentation', None))
+                gt_texts.append(ann.get('text_label'))
```

Comment:
```suggestion
                gt_texts.append(ann.get('text'))
```
`text` is concise and clear enough